### DISCUSSION TITLE
doesnt get deployed in heroku?

### DISCUSSION THREAD

**Discussion by Pasindu208** [phaticusthiccy/WhatsAsenaDuplicated#159] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/66366054/128290572-a4abfd91-ece3-4680-b923-5f0e2527378f.png)
hi,
please help me to make this work
https://github.com/udnisap208/KUMIKO-bot

thank you.

**Comment by phaticusthiccy** [comment, marked as answer]
Hi @udnisap208 

We don't support any fake bot. Also users shouldn't use fake bots. Like yours. Fake bots dosent have any cyber attack protection. It would be really dangerous for users. Please use original WhatsAsena. 

[Check our telegram group](https://t.me/AsenaSupport)

**Reply by Pasindu208** [reply]
Oh. Ok and thanks for the reply.
but what do you mean original bot?
I've only changed the appearance of the reply messages and added a few plugins

**Reply by phaticusthiccy** [reply]
This GitHub repository. If you have any doubts about the duplicated name, you can ask for information in [our telegram group](https://t.me/AsenaSupport) or you can read the description. 

If this repository once forked or cloned, the cyber attack protection will be closed. Every fake bots using our Docker. The cyber attack protection is there.

**Reply by Pasindu208** [reply]
Ok. Thank you  : )
